<launch>
  <!-- Load config parameters -->
  <rosparam command="load" file="$(find contact_control)/config/config_sia5.yaml" />

  <!-- moveit -->
  <arg name="sim" default="true"/>
  <arg name="sensor" default="false"/>
  <include file="$(find nrg_sia5_moveit_config)/launch/moveit_planning_execution.launch" />

  <!-- motion planning/control service (launch this separate) -->
  <!-- node name="contact_task_node" pkg="contact_task" type="contact_task_node" output="screen" respawn="false"/ -->

  <group ns="netft">
    <node name="netft_node" pkg="netft_rdt_driver" type="netft_node" respawn="false" output="screen" args="192.168.1.84" if="$(arg sensor)"/>
    <node name="netft_force_utils" pkg="netft_utils" type="netft_utils" respawn="false" output="screen" args="/base_link /eef" />
    <node name="netft_sim" pkg="netft_utils" type="netft_utils_sim" respawn="false" output="screen" unless="$(arg sensor)"/>
  </group>

</launch>
